pom.xml

```diff
@@ -30,14 +38,19 @@
       <version>${vertx.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>1.7.25</version>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>${log4j2.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>${log4j2.version}</version>
     </dependency>
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.17</version>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
Can you add a comment why SLF 1.7 API is used (log4j-slf4j-impl) and not SLF 1.8 API (log4j-slf4j18-impl)?
https://logging.apache.org/log4j/2.x/log4j-slf4j-impl/index.html

Reply:
@julianladisch 

I based this change on work I did months ago. I wasn't aware that the other library existed. Are you saying we should use `log4j-slf4j18-impl` instead?

Reply:
Java 11 uses the Java Module System and therefore requires SLF4J 1.8.x and log4j-slf4j18-impl. Using SLF 1.7 creates technical debt.